docs/UserGuide.md

```diff
@@ -29,11 +27,32 @@ Vax@NUS is a **one stop management app to efficiently track and schedule COVID-1
    * **`list`** : Lists all data.
    * **`exit`** : Exits the app.
 
-1. Refer to the [Features](#features) below for details of each command.
-
 --------------------------------------------------------------------------------------------------------------------
+## About
+
+This section aims to help you to navigate the user guide. 
 <br> 
 
+[Quick start](#quick-start) section has been provided to help you set up Vax@NUS. 
+<br>
+
+Alternatively, to learn more about our features and the details of each command, do refer to [Features](#features). 
+<br>
+
+If you are interested to know about the valid input taken in by our commands, do take a look at [Input-Formats](#input-formats).
+
+<div markdown="block" class="alert alert-info">
+
+:information_source: This icon indicates helpful notes. 
+
+</div>
+
+<div markdown="span" class="alert alert-primary">
+
+`command`: Words enclosed in a grey box are commands used by Vax@NUS' features. 
```

Comment:
In the `Input Formats` section, we also make things like `FACULTY` appear with grey background but they are not commands. Would "input into Vax@NUS" be a better alternative to "commands" here?

Reply:
I agree with Dan. Because 'command' is the word you are trying to explain, so I don't think you should use the same word in the explanation here

Reply:
Will it be confusing if we change it to "Input into Vax@NUS" instead of "commands" but we also have "Words in UPPER_CASE are the parameters to be supplied by the user." under command format?

Reply:
Alternatively, will it better if we change it to bolded words (instead of words) enclosed in grey box are commands used by Vax@NUS' features?